<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>468</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:#00bae2;</string>
   </property>
   <property name="text">
    <string>Lancement...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>378</y>
     <width>184</width>
     <height>56</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background:  url(:/ressources/download_play/install.png) no-repeat;
}

QPushButton:hover {
	background:  url(:/ressources/download_play/install_hover.png) no-repeat;
}</string>
   </property>
   <property name="iconSize">
    <size>
     <width>125</width>
     <height>127</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="newSmallImage1" native="true">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>132</y>
     <width>178</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background:   no-repeat;
	background-image: url(:/ressources/news/standard/news2.png)
}
</string>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="serverDescription" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>126</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  url(:/ressources/servers/sigma_description.png) no-repeat;</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundArtwork" native="true">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>38</y>
     <width>650</width>
     <height>385</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  url(:/ressources/artworks/artwork1.png) no-repeat;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newBigFront">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>128</y>
     <width>368</width>
     <height>212</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background: url(:/ressources/news/big/panel.png) no-repeat;
}

QPushButton:hover {
	background: url(:/ressources/news/big/panel_hover.png) no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBarTotal">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>458</y>
     <width>430</width>
     <height>8</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
	border: none;
	background-image: url(:/ressources/download_play/bar_empty.png);
	padding: 0px;
	color:white;
	font-size:8px;
}
QProgressBar::chunk {
	background-image: url(:/ressources/download_play/bar_full.png);
 }</string>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDownloadSpeed">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>438</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:#00bae2;</string>
   </property>
   <property name="text">
    <string>0 o/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="newSmallText2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>169</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white;
border: none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;News Title&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer eget sagittis tellus, eu lobortis nulla. Vestibulum non nunc tortor. Nulla rutrum condimentum euismod.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="newSmallFront2">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>128</y>
     <width>187</width>
     <height>212</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background: url(:/ressources/news/standard/panel.png) no-repeat;
}

QPushButton:hover {
	background: url(:/ressources/news/standard/panel_hover.png) no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="newBigImage" native="true">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>132</y>
     <width>357</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background:   no-repeat;
	background-image: url(:/ressources/news/big/news1.png);
}
</string>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="newSmallImage2" native="true">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>132</y>
     <width>178</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background:   no-repeat;
	background-image: url(:/ressources/news/standard/news3.png)
}
</string>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="newSmallText1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>169</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white;
border: none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;News Title&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer eget sagittis tellus, eu lobortis nulla. Vestibulum non nunc tortor. Nulla rutrum condimentum euismod.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="newSmallFront1">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>128</y>
     <width>187</width>
     <height>212</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background: url(:/ressources/news/standard/panel.png) no-repeat;
}

QPushButton:hover {
	background: url(:/ressources/news/standard/panel_hover.png) no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>408</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background:  url(:/ressources/download_play/settings.png) no-repeat;
}

QPushButton:hover {
	background:  url(:/ressources/download_play/settings_hover.png) no-repeat;
}</string>
   </property>
  </widget>
  <zorder>labelStatus</zorder>
  <zorder>playButton</zorder>
  <zorder>serverDescription</zorder>
  <zorder>backgroundArtwork</zorder>
  <zorder>progressBarTotal</zorder>
  <zorder>labelDownloadSpeed</zorder>
  <zorder>newBigImage</zorder>
  <zorder>newSmallImage2</zorder>
  <zorder>settingsButton</zorder>
  <zorder>newBigFront</zorder>
  <zorder>newSmallImage1</zorder>
  <zorder>newSmallFront1</zorder>
  <zorder>newSmallText1</zorder>
  <zorder>newSmallFront2</zorder>
  <zorder>newSmallText2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
